spec/features/filling_out_application_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+describe 'filling out an application' do
+  let(:application) do
+    create :application_template, :with_questions
+  end
+
+  before :each do
+    when_current_user_is :student, integration: true
+    visit application_path(application)
+  end
+
+  context 'answered all required questions' do
+    it 'redirects to the student dashboard' do
+      click_on 'Submit application'
+      expect(page.current_url).to eql student_dashboard_url
+    end
+
+    it 'renders a flash message' do
+      expect_flash_message(:application_receipt)
+      click_on('Submit application')
```

Comment:
Sorry, I meant to include the opposing viewpoint - a lot of people find parentheses a perfectly natural way of indicating the logical grouping of expressions, like in math, which is an even more widespread language than Ruby.

It's a good idea for readability's sake to keep the major points of code style consistent across a codebase, but this is such a minor point, that again, I don't think you need to go back and do anything about it. I just wanted to be informative. :smile: